Source/WebCore/html/FileInputType.cpp

```diff
@@ -152,12 +153,14 @@ FormControlState FileInputType::saveFormControlState() const
         stateVector.uncheckedAppend(AtomString { file->path() });
         stateVector.uncheckedAppend(AtomString { file->name() });
     }
+    stateVector.uncheckedAppend(AtomString { m_displayString });
```

Comment:
Is this `uncheckedAppend` safe?

From glancing at the code, it seems we've only reserved `(size * 2)` capacity in the vector, but we're now appending `1 + (size * 2)` items.